core/domain/email_manager_test.py

```diff
@@ -88,6 +89,131 @@ def test_sender_id_validation(self):
         # pylint: enable=protected-access
 
 
+class ExplortionMemebershipEmailTests(test_utils.GenericTestBase):
+    """ Tests that sending explortion membership email works as expected. """
```

Comment:
Ditto: spelling error. Also please remove spaces after first """ and before last """.